.appveyor.yml

```diff
@@ -1,41 +1,33 @@
+# Documentation: https://github.com/JuliaCI/Appveyor.jl
 environment:
   matrix:
-    - julia_version: 0.7
-    - julia_version: 1
+    - julia_version: 1.0
+    - julia_version: 1.3
     - julia_version: nightly
-
 platform:
-  - x86  # 32-bit
-  - x64  # 64-bit
-
-# # Uncomment the following lines to allow failures on nightly julia
-# # (tests will run but not make your overall status red)
-# matrix:
-#   allow_failures:
-#   - julia_version: nightly
-
+  - x86
+  - x64
+matrix:
+  allow_failures:
+    - julia_version: nightly
+    - platform: x86  # see https://github.com/invenia/JLSO.jl/issues/12
```

Comment:
Shouldn't we just delete this line since we're ignoring nightly failures on Travis altogether?